api/client/service/opts.go

```diff
@@ -176,61 +263,28 @@ func (m *MountOpt) Set(value string) error {
 		}
 	}
 
-	for _, field := range fields {
-		parts := strings.SplitN(field, "=", 2)
-		if len(parts) == 1 && strings.ToLower(parts[0]) == "writable" {
-			mount.Writable = true
-			continue
+	var err error
+	switch strings.ToLower(key) {
+	case "name":
```

Comment:
`name` doesn't really make sense for bind mounts, which is maybe ok.

By splitting these, it allows us to remove the `volume-` prefix on all of these other fields (below), which makes things a lot less verbose.